x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_type_menu.tsx

```diff
@@ -35,7 +36,16 @@ export const ActionTypeMenu = ({
   useEffect(() => {
     (async () => {
       try {
-        const availableActionTypes = actionTypes ?? (await loadActionTypes({ http }));
+        /**
+         * Hidden action types will be hidden only on Alerts & Actions.
+         * actionTypes prop is not filtered. Thus, any consumer that provides it's own actionTypes
+         * can use the hidden action types. For example, Cases or Detections of Security Solution.
+         *  */
```

Comment:
**nit:** I would also add to the comment the similar message that is used elsewhere

```
* TODO: Remove when cases connector is available across Kibana. Issue: https://github.com/elastic/kibana/issues/82502.
```